avocado/utils/pci.py

```diff
@@ -53,7 +53,7 @@ def get_pci_addresses():
     addresses = []
     cmd = "lspci -D"
     for line in process.system_output(cmd).splitlines():
-        if "PCI bridge" not in line:
+        if "PCI bridge" not in line and "peripheral" not in line:
```

Comment:
@ldoktor we have this entry in few new servers. I have found it on one of the server which I worked so added it. following is the output for your reference.
root@ltciofvtr-firestone1:~# lspci -D | grep -i peripheral
0003:01:00.1 System peripheral: PLX Technology, Inc. Device 87d0 (rev ca)
0003:01:00.2 System peripheral: PLX Technology, Inc. Device 87d0 (rev ca)
0003:01:00.3 System peripheral: PLX Technology, Inc. Device 87d0 (rev ca)
0003:01:00.4 System peripheral: PLX Technology, Inc. Device 87d0 (rev ca)
root@ltciofvtr-firestone1:~# 

root@ltc-briggs3:~# lspci -D | grep -i peripheral
0001:01:00.1 System peripheral: PLX Technology, Inc. Device 87d0 (rev ca)
0001:01:00.2 System peripheral: PLX Technology, Inc. Device 87d0 (rev ca)
0001:01:00.3 System peripheral: PLX Technology, Inc. Device 87d0 (rev ca)
0001:01:00.4 System peripheral: PLX Technology, Inc. Device 87d0 (rev ca)
root@ltc-briggs3:~#

from two different servers. hence I have added it to skip them. let me know if anything needed